spec.md

```diff
@@ -118,11 +118,12 @@ Every event in the OpenEventing specification includes context which is a set
 of consistent metadata attributes tools and developers can rely upon to
 understand how to handle the event and its data.
 
-Context is designed to be delivered separately from the event data (e.g. in
-protocol headers or protocol specific attributes).  This allows the context
-to be analyzed without having to deserialize the event data.  The context can
-also be serialized with the event data (e.g. a JSON implementation may
-consist of one JSON object that contains both context and data).
+Context is designed such that it can be delivered separately from the event data 
+(e.g. in protocol headers or protocol specific attributes).  This allows the context
+to be inspected at the destination without having to deserialize the event data.  
+The context may also need to be serialized with the event data for some use cases
```

Comment:
I know we haven't converted over to RFC2119 keywords yet, but we might as well start... can you change this "may" to a "MAY" ?  And the "may" on the next line to "might".